index.html

```diff
@@ -2590,26 +2586,25 @@ <h1>
     "type": "RsaVerificationKey2018",
     "controller": "did:example:123456789abcdefghi",
     "publicKeyPem": "-----BEGIN PUBLIC KEY...END PUBLIC KEY-----\r\n"
-  }, {
-    "id": "did:example:123456789abcdefghi#keys-2",
-    "type": "Ed25519VerificationKey2018",
-    "controller": "did:example:123456789abcdefghi",
-    "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
   }, {
     "id": "did:example:123456789abcdefghi#keys-3",
-    "type": "RsaPublicKeyExchangeKey2018",
+    "type": "ieee2410VerificationKey2018",
```

Comment:
Capitalize first letter of type:
```suggestion
    "type": "Ieee2410VerificationKey2018",
```

Reply:
Again, not my convention there -- just trying to keep things as close to how they were as before in this PR.